spring-kafka/src/main/java/org/springframework/kafka/event/ConcurrentContainerStoppedEvent.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2018-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.kafka.event;
+
+/**
+ * An event published when a container is stopped.
+ *
+ * @author Gary Russell
+ * @since 2.2
+ *
+ */
+public class ConcurrentContainerStoppedEvent extends KafkaEvent {
+
+	private static final long serialVersionUID = 1L;
+
+	/**
+	 * Construct an instance with the provided source and container.
+	 * @param source the container instance that generated the event.
+	 * @param container the container or the parent container if the container is a child.
+	 * @since 2.2.1
+	 */
+	public ConcurrentContainerStoppedEvent(Object source, Object container) {
+		super(source, container);
+	}
+
+	@Override
+	public String toString() {
+		return "ContainerStoppedEvent [source=" + getSource() + "]";
```

Comment:
Looks like this class is total copy/paste of existing.
Please, revise the intention of this class.

Reply:
I have updated ConcurrentContainerStoppedEvent as per the comments. Please review.

Reply:
Thanks.
There is no need to push the change and comment about it.
I just got to emails from you talking about same stuff: one is commit pushed to the PR and another your comment mentioning just pushed commit.
From my review perspective it is a duplication of the event happening on the PR.
So, if you agree with me, no need to say that you agree: just make a change and push it.

What is more important is to add more info into commit messages.
This way, over time we might extract the context of the change properly.

Reply:
Looks like this concern has not been addressed yet.